site/source/user-manual/service-guides/nextcloud/nextcloud-setup/nextcloud-linux.rst

```diff
@@ -55,7 +55,7 @@ You will first need to have the :ref:`Tor daemon running<tor-linux>`.
 
 1. On your desktop application. Click the account in the top left -> Settings, then in Settings, click Network, then "Specify proxy manually as" and "SOCKS5 proxy." Enter "127.0.0.1" for the Host and "9050" for the port.
 2. Close the Settings screen and click the account in the top left again, then "Add Account."
-3. On the following screen, click "Log in," then enter your Nextcloud Tor server address, which you can copy from the Nextcloud page on your Embassy -> Interfaces - Tor. This must start with `http://` and end with .onion. Click Next.
+3. On the following screen, click "Log in," then enter your Nextcloud Tor server address, which you can copy from the Nextcloud page on your Embassy -> Interfaces -> Tor. This must start with `http://` and end with `.onion`. Click Next.
```

Comment:
It's not this simple, but that's what my upcoming changes will be about so this is fine for now